hack/build.sh

```diff
@@ -61,7 +61,10 @@ run() {
     # Format source code
     go_fmt
```

Comment:
Minor point...

BTW, should we move `go_fmt` before compile and test? I hate the idea of the compiled and tested code being changed after.

Reply:
Good catch. We can do this, and as the --fast mode skips formatting anyway it doesn't matter whether it is skipped before or after compile ;-)